src/main/java/de/sakpaas/backend/v2/controller/UserController.java

```diff
@@ -15,9 +23,17 @@
 public class UserController {
```

Comment:
Arggg, entspricht nicht der openAPI Definition! Es müsste `/v2/users` sein!
https://docs.happyhamster.org/

Reply:
Als ich des geschrieben hab, gab es keine Doku den Endpunkt in der OpenApi nicht^^ Daher definiere ich die API als falsch